DOCS-421: base models v2 (#968)
viamrobotics/docs: docs/components/base/wheeled.md

---
title: "Configure a Wheeled Base"
linkTitle: "wheeled"
weight: 30
type: "docs"
description: "Configure a wheeled base."
tags: ["base", "components"]
# SMEs: Steve B
---

> A `wheeled` base is a model for a mobile robotic base with drive motors on both sides (differential steering).

Reviewer: ```suggestion
The `wheeled` base model supports basic mobile robotic bases with drive motors on both sides (differential steering).
```
[opt]
